integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyBucketsIntegrationTest.java

```diff
@@ -1002,6 +1005,373 @@ public void testLoanClassificationUsingAgeingArrears() {
         }
     }
 
+    @Test
+    public void testDelinquencyWithPauseLettingPauseExpire() {
```

Comment:
For one of this loans could you please enable installment level delinquency and check if the pause days are also taken into account there too?

Reply:
done